firewood/examples/insert.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE.md for licensing terms.
+
+use clap::Parser;
+use std::{error::Error, ops::RangeInclusive, sync::Arc, time::Instant};
+
+use firewood::{
+    db::{Batch, BatchOp, Db, DbConfig},
+    v2::api::{Db as DbApi, Proposal},
+};
+use rand::{distributions::Alphanumeric, Rng};
+
+#[derive(Parser, Debug)]
+struct Args {
+    #[arg(short, long, default_value = "1-64", value_parser = string_to_range)]
+    keylen: RangeInclusive<usize>,
+    #[arg(short, long, default_value = "32", value_parser = string_to_range)]
+    datalen: RangeInclusive<usize>,
+    #[arg(short, long, default_value_t = 1)]
+    batch_keys: usize,
+    #[arg(short, long, default_value_t = 100)]
+    inserts: usize,
+}
+
+fn string_to_range(input: &str) -> Result<RangeInclusive<usize>, Box<dyn Error + Sync + Send>> {
+    //<usize as std::str::FromStr>::Err> {
+    let parts: Vec<&str> = input.split('-').collect();
+    match parts.len() {
+        1 => Ok(input.parse()?..=input.parse()?),
+        2 => Ok(parts[0].parse()?..=parts[1].parse()?),
+        _ => Err("Too many dashes in input string".into()),
+    }
+}
```

Comment:
It's amazing how little code you need to be able to run this from the command line with all these options.